docker/env/build_n_push.sh

```diff
@@ -0,0 +1,32 @@
+# !/usr/bin/env bash
+set -e
+
+DOCKER_ENV_DIR=$(dirname $(readlink -f $0 ))
+VERSION=v$(cat ${DOCKER_ENV_DIR}/version)
```

Comment:
Using the sha for now is a bit problematic, since not everyone has the privilege to push into dockerhub scylladb/hydra. (don't get me wrong, i'm for granting everyone the permissions to push new versions)

in other places (i.e. dtest), the maintainer generate the image with timed versioned, and pass it along to whom is doing the PR to merge it with the relevant code.

Both aren't bullet proof...

Reply:
@bentsi what do you think ? should we go with timestamp or sha for the docker versions ?

Reply:
There is advantage for using hashes locally too: you will not interfere with code which will come from upstream during your own work. Imagine that somebody bumped hydra version while you trying to do the same in your branch.